lib/mix/nerves/shell.ex

```diff
@@ -7,7 +7,7 @@ defmodule Mix.Nerves.Shell do
     # We use the tty_sl driver for input because it handles tty geometry and
     # streaming mode.
     stdin_port = Port.open({:spawn, "tty_sl -c -e"}, [:binary, :eof, :stream, :in])
-
+    Application.stop(:logger)
```

Comment:
Is the `Application.stop(:logger)` needed?

Reply:
Logger errors out when the user_dir is switched. Stopping logger prevents the console from writing logger errors to the console.

Reply:
Stopping logger prevents it from printing error messages about `user_drv` being used.